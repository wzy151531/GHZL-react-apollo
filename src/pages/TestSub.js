import React from 'react';

export const TestSub = () => {
  return (
    <div>TestSub.<p>This page can only be accessed by admin.</p></div>
  );
};