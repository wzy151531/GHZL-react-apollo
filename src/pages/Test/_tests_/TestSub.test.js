import React from 'react';
import TestSub from '../TestSub';
import renderer from 'react-test-renderer';

test('TestSub should render correctly', () => {
  const component = renderer.create(
    <TestSub/>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});