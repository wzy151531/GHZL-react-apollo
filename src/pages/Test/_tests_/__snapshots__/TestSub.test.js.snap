// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestSub should render correctly 1`] = `
<div>
  TestSub.
  <p>
    This page can only be accessed by admin.
  </p>
</div>
`;
